tests/unit/test_stas.py

```diff
@@ -0,0 +1,21 @@
+from modules.data import PostData
```

Comment:
This file is named `test_stas.py` instead of `test_stats.py`? Can you rename it pls? :D

Reply:
done, thank you